fcrepo-kernel-impl/pom.xml

```diff
@@ -27,6 +27,7 @@
     <dependency>
       <groupId>org.modeshape</groupId>
       <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
I have the same confusion about this scope as above. I guess I just don't understand, given our disengagement with J2EE or OSGi, what we actually mean by `provided`...